pkg/reader/usdc_reader.go

```diff
@@ -120,14 +129,34 @@ func (u usdcMessageReader) MessageHashes(
 		return nil, fmt.Errorf("no contract bound for chain %d", source)
 	}
 
+	eventFilter := make([]query.Expression, 0, len(eventIDs))
+	for _, id := range eventIDs {
+		eventFilter = append(
+			eventFilter,
+			query.Comparator(consts.EventFilterCCIPMessageSent,
```

Comment:
Name for value filter shouldn't be mixed up with the key name. I would prefer if the comparatror was called something like MessageValue

Reply:
I don't follow. Is it about the name of the const (`consts.EventFilterCCIPMessageSent`) or its value? What do you suggest instead?

> Name for value filter primitive shouldn't be mixed up with the key name.
I think it's different, isn't it?
Query key is `consts.EventNameCCTPMessageSent`, but comparator is `consts.EventFilterCCIPMessageSent`, am I missing sth?

Reply:
Sry I misread that multiple times while looking at the code. Still, wouldn't constants like this: `consts.CCIPMessageSentValue `and for the event itself `consts.CCIPMessageSent` make more sense?

Reply:
Trying to follow existing patters in naming, so I've renamed to
```go
consts.CCTPMessageSentValue # for the comparator
consts.EventNameCCTPMessageSent # kept existing for the event
```

Reply:
sgtm